packages/thirdweb/src/exports/utils.ts

```diff
@@ -132,6 +132,7 @@ export {
 // abi
 // ------------------------------------------------
 export { encodeAbiParameters } from "../utils/abi/encodeAbiParameters.js";
+export { encodePacked } from "viem";
```

Comment:
What's this for?

Reply:
re-exported as a util because eiman had a case where he needed it to encode some contract call, thought it was useful to have